cclient/serialization/json/find_transactions.c

```diff
@@ -58,30 +70,58 @@ retcode_t json_find_transactions_deserialize_request(serializer_t const* const s
   retcode_t ret = RC_OK;
   cJSON* json_obj = cJSON_Parse(obj);
   cJSON* json_item = NULL;
+  bool empty = true;
 
   log_debug(json_logger_id, "[%s:%d] %s\n", __func__, __LINE__, obj);
   JSON_CHECK_ERROR(json_obj, json_item, json_logger_id);
 
   ret = json_array_to_hash243_queue(json_obj, "bundles", &out->bundles);
```

Comment:
Can you use `cJSON_HasObjectItem` to check if it's present or not ?

Reply:
Please use cJSON_HasObjectItem here instead of relying on the return value of `json_array_to_hash243_queue `